### DISCUSSION TITLE
Twilio CDN cache policy

### DISCUSSION THREAD

**Discussion by amPerl** [twilio-labs/paste#1717] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/461652/127687686-d5b98b78-9139-4c80-a3f3-268a1846cae5.png)

Paste docs suggest pre-connecting to the Twilio CDN for quicker asset loads. This appears to work well (better than loading on JS init, of course!) but Lighthouse reports give a big red blip for poor cache headers on the resources themselves.

![image](https://user-images.githubusercontent.com/461652/127687870-2bebd30e-92fc-42e8-8519-97382698eabd.png)

Is this something the Paste team could remedy? Are there any alternative solutions to promote better caching?

**Comment by SiTaggart** [comment, marked as answer]
Hi @amPerl 

This seems to be a problem with the Twilio asset CDN which I've raised previously back in Decemeber with the Edge team that own it.

Feel free to jump on here too https://twilio.slack.com/archives/C1HK39003/p1608225601063100
